docs/src/dataDisplay/comment/comment.jsx

```diff
@@ -149,7 +149,7 @@ export default class CommentSample extends React.Component {
         });
     }
 
-    _onMenuClick(id, menuIsOpen) {
+    onMenuClick(id, menuIsOpen) {
         console.log(\`Action menu for comment \$\{id\} clicked!  Menu open: \$\{menuIsOpen\}\`);
```

Comment:
_Not Blocking._  We might consider ESLINT suppression here for the `console.log()`.

Reply:
Now that I realize that this is in the sample source code string constant, it's _**even more** non-blocking_. 😀

Reply:
me too